src/Main/Main.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Props = {
+  children: JSX.Element;
+};
+
+export const Main: React.FC<Props> = ({ children }) => (
+  <section
+    className="todoapp__main"
+  >
```

Comment:
If the tag has less than 3 attributes and they fit in a line of up to 80 characters:
```suggestion
  <section className="todoapp__main">
```